sections/exchange/FooterCard/TradeSummaryCard/GasPriceSummaryItem.tsx

```diff
@@ -31,8 +31,10 @@ const GasPriceSummaryItem: FC<GasPriceSummaryItemProps> = ({
 }) => {
 	const { t } = useTranslation();
 	const { selectedPriceCurrency } = useSelectedPriceCurrency();
-	const isL2 = useRecoilValue(isL2State);
-	const isMainnet = useRecoilValue(isMainnetState);
+	const { chain: network } = useNetwork();
+	const isL2 = useIsL2();
+	const isMainnet =
+		network !== undefined ? [chain.mainnet.id, chain.goerli.id].includes(network?.id) : false;
```

Comment:
is this meant to be isL1 as goerli is a testnet and if so could use the useIsL1 hook as it's the same check?

Reply:
Yes. This is supposed to use `useIsL1` hook.